### ISSUE TITLE
Rendering to file

### ISSUE THREAD

**Issue by fekga** [markusfisch/ShaderEditor#25]
It would be a nice feature to render the shader output to a file (only an image, or even a video).
It is possible to capture the output with screenshot or screen recording but with low-fps shaders the video will be lagging.

**Comment by markusfisch** [comment]
Good idea. Images are probably simple because the thumbnails are generated from shader output already. Have a look at [ShaderRenderer::saveThumbnail()](https://github.com/markusfisch/ShaderEditor/blob/master/ShaderEditor/src/main/java/de/markusfisch/android/shadereditor/opengl/ShaderRenderer.java#L737). So that would require _only_ some UI.

To capture video output, an additional library would be required (like jcodec). And it probably will have an impact on performance too. But it's still a nice idea.

**Comment by markusfisch** [comment]
Oh, I just learned (MediaMuxer)[http://developer.android.com/reference/android/media/MediaMuxer.html] is available from API level 18. That may be an interesting option too.

**Comment by fekga** [comment]
That's great news! It would be nice to set a length for the rendering too (seconds or frames).

**Comment by fekga** [comment]
How is this issue going? Will there be any news on this?It would be a really nice feature!

**Comment by markusfisch** [comment]
Yes, I know, but unfortunately I didn't had the time yet. It's still on the list.
